tools/install.py

```diff
@@ -133,7 +133,8 @@ def files(action):
       if sys.platform != 'darwin':
         output_prefix += 'lib.target/'
 
-  action([output_prefix + output_file], 'bin/' + output_file)
+  action([output_prefix + output_file], (
+    'bin/' if 'false' == variables.get('node_shared') else 'lib/') + output_file)
```

Comment:
Will be easier on the eyes when written as an if/else statement.  That's also the commonest style in this file and will also let you keep it <= 80 columns.

Reply:
Like this?
```py
if 'false' == variables.get('node_shared'):
  action([output_prefix + output_file], 'bin/' + output_file)
else:
 action([output_prefix + output_file], 'lib/' + output_file)
```

Reply:
Ping @bnoordhuis

Reply:
@yorkie that looks like what Ben was suggesting, could you update this branch with that code?